bridge-history-api/internal/controller/history_controller.go

```diff
@@ -63,10 +82,38 @@ func (c *HistoryController) PostQueryTxsByHash(ctx *gin.Context) {
 		types.RenderFailure(ctx, types.ErrParameterInvalidNo, err)
 		return
 	}
-	result, err := c.historyLogic.GetTxsByHashes(ctx, req.Txs)
-	if err != nil {
-		types.RenderFailure(ctx, types.ErrGetTxsByHashFailure, err)
-		return
+
+	const cacheKeyPrefix = "queryTxsByHash:"
```

Comment:
Move out from this function.

Reply:
changed in [850be1e](https://github.com/scroll-tech/scroll/pull/966/commits/850be1e6667441fc8f118dac43603f14a97fd691)